Sources/TuistCore/Cache/CacheOutputType.swift

```diff
@@ -8,17 +14,65 @@ public enum CacheOutputType: CustomStringConvertible {
     /// Frameworks built for the simulator.
     case framework
 
-    /// XCFrameworks built for the simulator and device.
-    case xcframework
+    /// XCFrameworks built for the simulator and/or device.
+    case xcframework(CacheXCFrameworkDestination?)
```

Comment:
We should use an OptionSet instead of a nullable

Reply:
Updated to use `OptionSet`, to parse destination from commandline argument I need to create a initilization from String, pls take a look, I don't really sure if that init is good enough.